tasties_app/views.py

```diff
@@ -61,3 +61,14 @@ def register(request):
 
     context = {'form': form}
     return render(request, 'tasties_app/register.html', context)
+
+
+@login_required(login_url='login')
+def single_recipe(request, recipe=None):
+    if not recipe:
+        recipe = Recipe.objects.first()
+    ingredients = Ingredient.objects.filter(recipe_id=recipe)
```

Comment:
Another thing that will need to be in here in the future is automatic/selective unit conversion. For example, when we exceed some "reasonable" value, let's say we have "1052 grams" of some ingredient, we may convert to "1.052 Kilograms", we might as well do it in the view, since we may also want to implement some "Metric/Imperial" switch - should we decide to pursue that.

Reply:
As for this comment and the one above - At this moment I doubt that we will have time for that.
Let's continue with the tasks we agreed on for now.
If we'll finish earlier than expected we could try adding that conversion functionality.